src/Xaki.AspNetCore/TagHelpers/LocalizedEditor.cs

```diff
@@ -129,22 +129,22 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
 
                         (function () {{
                             var getMaxWidth = function(elements) {{
-                                var width = 0;
```

Comment:
Yeah the only reason I use `var` over `let` is for older browser support (I guess IE 10), will need to decide if I care about IE 10 but maybe it's time to move on.

Reply:
`Let` IE go.